packages/mobile/views/dashboard/DashboardRouter.svelte

```diff
@@ -1,14 +1,24 @@
 <script lang="typescript">
     import features from '../../features/features'
-    import { dashboardRoute, DashboardRoute, dashboardRouter, sendRouter } from '../../lib/routers'
+    import {
+        accountSwitcherRouter,
+        dashboardRoute,
+        DashboardRoute,
+        dashboardRouter,
+        sendRouter,
+    } from '../../lib/routers'
     import { AccountSwitcherDrawer, ReceiveDrawer, SendDrawer } from './drawers'
 
     function onReceiveDrawerClose(): void {
         $dashboardRouter.previous()
     }
     function onSendDrawerClose(): void {
-        $dashboardRouter.previous()
         $sendRouter.reset()
+        $dashboardRouter.previous()
```

Comment:
I changed the sequence here since I realised the drawer sometimes gets bigger for a small amount of time during the close animation. For instance with the create wallet view

Reply:
I also have this in my PR so perfect 👍🏼